### PR TITLE
Bug 1314317 - Generalize e10s dashboard for future comparisons

### PR DESCRIPTION
**Pull request by rwood-moz**

### CODE REVIEW

File: ui/js/controllers/perf/dashboard.js [hunk 1]
```diff
@@ -190,28 +209,36 @@ perf.controller('e10sCtrl', [
...
         });
         $scope.revision = $stateParams.revision;
+        $scope.topic = $stateParams.topic;
+        $scope.dashboardValues = phDashboardValues[$scope.topic];
+        $scope.variantDataOpt = $scope.dashboardValues['variantDataOpt'];
+        $scope.framework = $scope.dashboardValues['framework'];
+        $scope.header = $scope.dashboardValues['header'];
+        $scope.descP1 = $scope.dashboardValues['descP1'];
+        $scope.baseTitle = $scope.dashboardValues['baseTitle'];
+        $scope.variantTitle = $scope.dashboardValues['variantTitle'];
```
Comment (wlach) [inline ui/js/controllers/perf/dashboard.js]: Maybe better to do something like:

```js
['variantDataOt', 'framework', ...].forEach(function(k) {
  $scope[k] = phDashboardValues[$scope.topic][k];
});

File: ui/js/controllers/perf/dashboard.js [hunk 2]
```diff
@@ -190,28 +209,36 @@ perf.controller('e10sCtrl', [
...
 
         $scope.dataLoading = true;
         $scope.timeRanges = phTimeRanges;
         $scope.selectedTimeRange = _.find(phTimeRanges, {
-            value: ($stateParams.timerange) ? parseInt($stateParams.timerange) : e10sDefaultTimeRange
+            value: ($stateParams.timerange) ? parseInt($stateParams.timerange) : defaultTimeRange
         });
         $scope.revision = $stateParams.revision;
+        $scope.topic = $stateParams.topic;
+        $scope.dashboardValues = phDashboardValues[$scope.topic];
```
Comment (wlach) [inline ui/js/controllers/perf/dashboard.js]: Do we actually need this scope variable?

Comment (rwood-moz) [inline ui/js/controllers/perf/dashboard.js]: Ah yes good point

File: ui/js/controllers/perf/dashboard.js (hunk 1 above)
Comment (rwood-moz) [inline ui/js/controllers/perf/dashboard.js]: Yes that's more tidy, thanks!

**Comment by rwood-moz** [comment]
Tried to rebase and fix the conflict but completely messed up my branch, closing this and will open a new PR.

**Pull request closed without merging**